package.json

```
@@ -82,6 +82,12 @@
       "module": "./esm/urql.js",
       "import": "./esm/urql.mjs",
       "default": "./urql.js"
+    },
+    "./babel-plugin/debug-label": {
+      "types": "./babelPluginDefaultLabel.d.ts",
+      "module": "./esm/babelPluginDefaultLabel.js",
+      "import": "./esm/babelPluginDefaultLabel.mjs",
+      "default": "./babelPluginDefaultLabel.js"
```

As I tried, with nextjs on codesandbox, it doesn't seem to understand "exports". So, can we keep the file structure for old bundlers/envs?
```suggestion
    "./babel-plugin-debug-label": {
      "types": "./babel-plugin-debug-label.d.ts",
      "module": "./esm/babel-plugin-debug-label.js",
      "import": "./esm/babel-plugin-debug-label.mjs",
      "default": "./babel-plugin-debug-label.js"
```

Or, we can use `babelPluginDebugLabel` instead if we prefer camelCase.

---

I actually wanted to restructure it to `./babel/plugin-debug-label`, because it would align with `./babel/plugin-react-refresh` and `./babel/preset`. If we want to support old bundlers for this, we will be "splitting" the `babel` packages.

---

`./babel/plugin-debug-label` looks nice!

Please patch rollup.confg.js:
```diff
diff --git a/rollup.config.js b/rollup.config.js
index 1a931ec..1c9e589 100644
--- a/rollup.config.js
+++ b/rollup.config.js
@@ -72,7 +72,7 @@ function createCommonJSConfig(input, output) {
 export default function (args) {
   let c = Object.keys(args).find((key) => key.startsWith('config-'))
   if (c) {
-    c = c.slice('config-'.length)
+    c = c.slice('config-'.length).replaceAll('_', '/')
     return [
       createCommonJSConfig(`src/${c}.ts`, `dist/${c}.js`),
       createESMConfig(`src/${c}.ts`, `dist/esm/${c}.mjs`),
```

and then:
```suggestion
    "./babel/plugin-debug-label": {
      "types": "./babel/plugin-debug-label.d.ts",
      "module": "./esm/babel/plugin-debuglabel.js",
      "import": "./esm/babel/plugin-debuglabel.mjs",
      "default": "./babel/plugin-debuglabel.js"
```